examples/tutorial/jupyter/test/test_notebooks.py

```diff
@@ -92,29 +97,52 @@ def _replace_str(nb, original_str, str_to_replace):
     ].replace(original_str, str_to_replace)
 
 
+def _get_nb_dir(execution):
```

Comment:
Taking into account OmniSci we should have separate tests for each execution.

Reply:
But why? As for now OmniSci notebooks seems to me very similar to Ray/Dask notebooks. I think we can use the single test file as it is done now in this PR and avoid unnecessary duplication.

Reply:
Not sure OmniSci notebooks will seem to be very similar to Ray/Dask until #4001 is ready to merge. I would prefer separate them for now and combine when we have those in master as part of a new PR.

Reply:
Ok, let's use unique tests for each of the executions.